modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -295,41 +341,77 @@ How much the buffer size expands if it exceeds its initial size. A value of zero
-| Message Id a| String | Unique identifier for the message | None | Yes
-| Properties a| Object | Custom properties associated with the message | None | No
-| Protocol a| String | The protocol the Anypoint MQ server uses | None | No
-| Server Name a| String | The name of the Anypoint MQ server that processes the message | None | Yes
-|======================
+| Ack Token | String | Token used for acknowledging the receipt of a message | None | No
+| Channel Name | String | The name of the Anypoint MQ channel where the message is published or consumed | None | Yes
+| Channel Parameters | Object | Additional parameters or settings specific to the channel | None | No
+| Content Type | String | The MIME type of the message content | None | No
+| Delivery Count | Number | The number of times the message is delivered | None | No
+| Headers | Object | Metadata associated with the message, typically key-value pairs | None | No
+| Message Id | String | Unique identifier for the message | None | Yes
+| Properties | Object | Custom properties associated with the message | None | No
+| Protocol | String | The protocol the Anypoint MQ server uses | None | No
+| Server Name | String | The name of the Anypoint MQ server that processes the message | None | Yes
+|===
+
+[[SolaceMessageAttributes]]
+=== Solace Message Attributes
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|===
+| Field | Type | Description | Default Value | Required
+| DMQ Eligible | Boolean | Indicates if the message is eligible for Dead Message Queue | `False` | No
+| Channel Name | String | The name of the Solace channel where the message is published or consumed | None | No
+| Channel Parameters | Object | Additional parameters or settings specific to the Solace channel | None | No
+| Content Type | String | The MIME type of the message content | None | No
+| Delivery Mode | String | The delivery mode of the message | None | No
+| Discard Indication | Boolean | Indicates if the message is discarded | `False` | No
+| Eliding Eligible | Boolean | Indicates if the message is eligible for eliding | `False` | No
+| Endpoint Type | String | The type of endpoint used | None | No
+| Message Reference Id | String | A reference identifier for the message | None | No
+| Priority | Number | The priority level of the message | 0 | No
+| Protocol | String | The protocol the Solace server uses | None | No
+| Receive Timestamp | DateTime | The timestamp when the message was received | None | No
+| Redelivered | Boolean | Indicates if the message is redelivered | `False` | No
+| Reply | Boolean | Indicates if the message expects a reply | `False` | No
+| Server Name | String | The name of the Solace server that processes the message | None | Yes
+| Time To Live | Number | The time in milliseconds that the message remains valid | 0 | No
+|===
+
+[[SalesforcePubSubAttributes]]
+=== Salesforce Pub Sub Attributes
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
```

Comment:
Should the table specs match for all the tables in this topic?

Reply:
Sure! Great catch! Thanks!